### PR TITLE
Add the new product page

### PR DESCRIPTION
**Pull request by Ar-ali0601**
please review the code

### CODE REVIEW

File: src/main/java/pomPages/ProductPage.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package pomPages;
+
+public class ProductPage {
+	
+	/*
+	 * element to clickLink
```
Comment (aliraza61199) [inline src/main/java/pomPages/ProductPage.java:6]: this page is no available in  this application

**Review by aliraza61199 — COMMENTED** [review]
remove this page

**Comment by Ar-ali0601** [comment]
please review my code

File: src/main/java/pomPages/ProductPage.java (hunk 1 above)
Comment (Ar-ali0601) [reply src/main/java/pomPages/ProductPage.java]: i fixed the review comment

**Pull request merged**